app/src/main/kotlin/com/x8bit/bitwarden/ui/platform/components/dialog/BitwardenOverwritePasswordConfirmationDialog.kt

```diff
@@ -0,0 +1,30 @@
+package com.x8bit.bitwarden.ui.platform.components.dialog
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.bitwarden.ui.platform.components.dialog.BitwardenTwoButtonDialog
+import com.bitwarden.ui.platform.resource.BitwardenString
+
+/**
+ * A reusable dialog for confirming whether or not the user wants to overwrite an existing password
+ * credential.
+ *
+ * @param onConfirmClick A callback for when the overwrite confirmation button is clicked.
+ * @param onDismissRequest A callback for when the dialog is requesting dismissal.
+ */
+@Suppress("MaxLineLength")
+@Composable
+fun BitwardenOverwritePasswordConfirmationDialog(
```

Comment:
Can we modify `BitwardenOverwritePasskeyConfirmationDialog` to either accept the title and message and rename it to `BitwardenOverwriteCredentialConfirmationDialog`? That will make it more flexible if we add support for more credential types.